spec/gate/indexSpec.js

```diff
@@ -89,4 +89,27 @@ describe('UserGate', function() {
       expect(gate.allows('bar@mixmax.com')).toBe(true);
     });
   });
+
+  describe('uniform sample', () => {
+    it('should be deterministic', () => {
+      const gate = new UserGate({ sample: 0.5 });
+      expect(gate._matchesUniformSample('alice@mixmax.com')).toBe(true);
```

Comment:
Can we test determinism over greater iterations? Perhaps like [this](https://github.com/mixmaxhq/emailer-chrome-extension-source/blob/fd32ee26cf1a8d0d7a5642a3ef4b889fbdf7f9d8/test/spec/utils/TrialUtilsSpec.js#L45)

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/bbaee3d607cd7dc45dd72ebe13de72fb55c7c6d1